cmd/lakectl/cmd/fs_download.go

```diff
@@ -18,6 +18,11 @@ import (
 	"github.com/treeverse/lakefs/pkg/uri"
 )
 
+type objectInfo struct {
+	relPath string
```

Comment:
relative to what? document

Reply:
added comment.

relative to the source path in the repo - used the same terminology used on lakectl fs download code.